### ISSUE TITLE
ysi.so

### ISSUE THREAD

**Issue by Y-Less** [pawn-lang/YSI-Includes#102]
Apparently that domain exists - I entirely forgot about that until I got an e-mail just now!  Maybe should use that for something...

**Comment by Southclaws** [comment]
[aint.ysi.so/useful](aint.ysi.so/useful)

But in all seriousness, could redirect it to the wiki then include the URL in YSI error messages. Or (if there's a server available, or github pages might work) make error codes/common errors link to issues/solutions (ysi.so/101 would link here for example).

Also, crazy idea that would probably be a waste of time but I've always wanted to attempt: package manager for SA:MP, like Python's `pip` or Ruby's `gem`...

**Comment by Ahmad45123** [comment]
I agree about the redirect to the wiki, however, the wiki is not even good.. Its just a replicate to the PAWNDoc... There should be a more proper explanation to each library as well I guess. (I'll try to do some when I am free).

And @Southclaw, I tried to do some type of a package manager in my ExtremeStudio project but no one seem to really like it.. Or actually no one likes the whole IDE I guess XD...

**Comment by Southclaws** [comment]
If I ever did work on a package manager, it would be a completely standalone binary that could be plugged into any IDE/editor. But adoption would probably be nonexistent considering the SA:MP development community seems to be drying up (Yeah it sounds cliché but I just haven't seen any really interesting releases or projects on the forums for a long time!)

**Comment by Y-Less** [comment]
I don't know if its a good domain (actually, I kind of hate it, but ysi.tl went up drastically in price and I wasn't prepared to pay the hike), just saying it is there apparently.

**Comment by Crayder** [comment]
```
I agree about the redirect to the wiki, however, the wiki is not even good.. Its just a replicate to the PAWNDoc... There should be a more proper explanation to each library as well I guess. (I'll try to do some when I am free).
```

Out of most people you would know that this is already planned. Also, Southclaw agreed to helping with it.

I just haven't had the time recently and I planned on doing it as soon as Summer came. I still haven't added the param details even though we fixed the exporter, I'd have to go through and sort the files all over again to do so right.

**Comment by PT-Player** [comment]
@Y-Less what you think about samp-ysi.com ?

**Comment by Y-Less** [comment]
https://www.youtube.com/watch?v=54Mp5w6izO0
